iron-doc-nav.html

```diff
@@ -117,7 +137,22 @@
           if (!name) {
             continue;
           }
-          sectionItems.push({name: name, path: pathPrefix + name});
+          var path = pathPrefix + name;
+
+          var demos = [];
+          for (var d = 0; d < (item.demos || []).length; d++) {
+            var demo = item.demos[d];
+            if (demo.url && demo.description) {
+              var shortName =
+                  demo.description.toLowerCase().replace(/\s+/g, '-');
+              demos.push({
+                path: path + '/demos/' + shortName,
+                description: demo.description
+              });
```

Comment:
Sometimes they have demos with no description, in which case the path should probably just be used.